const e=JSON.parse('{"key":"v-d83e141a","path":"/testnote/Pytest%E7%B3%BB%E5%88%97%EF%BC%8805%EF%BC%89---%E6%B5%8B%E8%AF%95%E7%94%A8%E4%BE%8B%E6%89%A7%E8%A1%8C%E5%90%8E%E7%9A%84%E5%87%A0%E7%A7%8D%E7%8A%B6%E6%80%81.html","title":"","lang":"zh-CN","frontmatter":{"description":"用例执行状态 用例执行完成后，每条用例都有自己的状态，常见的状态有 passed：测试通过; failed：断言失败; error：用例本身写的质量不行，本身代码报错（譬如：fixture不存在，fixture里面有报错）; xfail：预期失败，加了 @pytest.mark.xfail(); error的栗子一：参数不存在 为啥是error pwd...","head":[["meta",{"property":"og:url","content":"https://vuepress-theme-hope-docs-demo.netlify.app/gbook/testnote/Pytest%E7%B3%BB%E5%88%97%EF%BC%8805%EF%BC%89---%E6%B5%8B%E8%AF%95%E7%94%A8%E4%BE%8B%E6%89%A7%E8%A1%8C%E5%90%8E%E7%9A%84%E5%87%A0%E7%A7%8D%E7%8A%B6%E6%80%81.html"}],["meta",{"property":"og:site_name","content":"文档演示"}],["meta",{"property":"og:description","content":"用例执行状态 用例执行完成后，每条用例都有自己的状态，常见的状态有 passed：测试通过; failed：断言失败; error：用例本身写的质量不行，本身代码报错（譬如：fixture不存在，fixture里面有报错）; xfail：预期失败，加了 @pytest.mark.xfail(); error的栗子一：参数不存在 为啥是error pwd..."}],["meta",{"property":"og:type","content":"article"}],["meta",{"property":"og:locale","content":"zh-CN"}],["meta",{"property":"og:updated_time","content":"2023-07-03T09:19:59.000Z"}],["meta",{"property":"article:author","content":"gstorms"}],["meta",{"property":"article:modified_time","content":"2023-07-03T09:19:59.000Z"}],["script",{"type":"application/ld+json"},"{\\"@context\\":\\"https://schema.org\\",\\"@type\\":\\"Article\\",\\"headline\\":\\"\\",\\"image\\":[\\"\\"],\\"dateModified\\":\\"2023-07-03T09:19:59.000Z\\",\\"author\\":[{\\"@type\\":\\"Person\\",\\"name\\":\\"gstorms\\",\\"url\\":\\"https://gtcool.top\\"}]}"]]},"headers":[{"level":2,"title":"用例执行状态","slug":"用例执行状态","link":"#用例执行状态","children":[]},{"level":2,"title":"error的栗子一：参数不存在","slug":"error的栗子一-参数不存在","link":"#error的栗子一-参数不存在","children":[{"level":3,"title":"为啥是error","slug":"为啥是error","link":"#为啥是error","children":[]}]},{"level":2,"title":"error的栗子二：fixture有错","slug":"error的栗子二-fixture有错","link":"#error的栗子二-fixture有错","children":[{"level":3,"title":"为啥是error？","slug":"为啥是error-1","link":"#为啥是error-1","children":[]}]},{"level":2,"title":"failed的栗子一","slug":"failed的栗子一","link":"#failed的栗子一","children":[{"level":3,"title":"为啥是failed","slug":"为啥是failed","link":"#为啥是failed","children":[]}]},{"level":2,"title":"failed的栗子二","slug":"failed的栗子二","link":"#failed的栗子二","children":[{"level":3,"title":"为啥是failed","slug":"为啥是failed-1","link":"#为啥是failed-1","children":[]}]},{"level":2,"title":"总结","slug":"总结","link":"#总结","children":[]},{"level":2,"title":"xfail的栗子","slug":"xfail的栗子","link":"#xfail的栗子","children":[{"level":3,"title":"为啥是xfail","slug":"为啥是xfail","link":"#为啥是xfail","children":[]}]}],"git":{"createdTime":1688375999000,"updatedTime":1688375999000,"contributors":[{"name":"gstorms","email":"wangzx_2015@foxmail.com","commits":1}]},"readingTime":{"minutes":1.48,"words":445},"filePathRelative":"testnote/Pytest系列（05）---测试用例执行后的几种状态.md","localizedDate":"2023年7月3日","autoDesc":true,"excerpt":""}');export{e as data};
